helper/hex/hex.go

```diff
@@ -95,5 +95,9 @@ func DecodeHexToBig(hexNum string) *big.Int {
 }
 
 func DropHexPrefix(input []byte) []byte {
-	return input[2:]
+	if len(input) >= 2 && strings.ToLower(string(input[:2])) == "0x" {
+		return input[2:]
+	}
+
+	return input
```

Comment:
Just dropping as an idea, not sure what is better:
```suggestion
	return []byte(strings.TrimPrefix(strings.ToLower(string(input)), "0x"))
```
What I don't like with my proposal is casting back and forth to string and back to a byte slice. Although proposed it anyways because of sparing some lines of code and checks :slightly_smiling_face: 

You decide what makes more sense to you.

Reply:
I prefer this one-liner, and dropping `DropHexPrefix`

Reply:
Fixed as Victor suggested 031bb22